lib/Client.php

```diff
@@ -349,9 +356,9 @@ private function createCurlOptions($method, $body = null, $headers = null)
         $options = [
                 CURLOPT_RETURNTRANSFER => true,
                 CURLOPT_HEADER => true,
-                CURLOPT_CUSTOMREQUEST => strtoupper($method),
+                CURLOPT_CUSTOMREQUEST => mb_strtoupper($method),
```

Comment:
If we want to use the `mb_strtoupper`, we have to edit `composer.json` and add the `ext-mbstring: "*"` in `require` block.